core/frontend/src/components/kraken/ExtensionModal.vue

```diff
@@ -0,0 +1,200 @@
+              outlined
+              width="100%"
+            >
+              <v-card-text
+                style="overflow: auto;"
+              >
+                <pre>{{ permissions }}</pre>
+              </v-card-text>
+            </v-card>
+            <span v-if="selected?.company">
+              <v-card
+                class="mt-2"
+                outlined
+              >
+                <span
+                  class="mb-3 align-center"
+                  style="display: flex; flex: 1 1 auto;"
+                >
+                  <v-img
+                    class="ma-2"
+                    :src="extension.company_logo"
+                    max-height="30"
+                    max-width="30"
+                  />
+                  <h3 class="ma-2">
+                    {{ selected?.company.name }}
+                  </h3>
+                </span>
+                <v-card-text v-if="selected?.company.about">
+                  {{ selected?.company.about }}
+                </v-card-text>
+                <v-card-text>
+                  Written by:
+                  <ul>
+                    <li
+                      v-for="author in (selected?.authors ?? [])"
+                      :key="author.email"
+                    >
+                      {{ author.name }} &#60;{{ author.email }}>
+                    </li>
+                  </ul>
+                </v-card-text>
+              </v-card>
+            </span>
+          </v-sheet>
+        </v-col>
+      </v-row>
+    </v-card-text>
+  </v-card>
+</template>
+
+<script lang="ts">
+import { marked } from 'marked'
+import Vue, { PropType } from 'vue'
+
+import { ExtensionData, Version } from '@/types/kraken'
+
+export default Vue.extend({
+  name: 'ExtensionModal',
+  props: {
+    extension: {
+      type: Object as PropType<ExtensionData>,
+      required: true,
+    },
+    installed: {
+      type: String,
+      default: null,
+      required: false,
+    },
+  },
+  data() {
+    return {
+      selected_version: '' as string,
+    }
+  },
+  computed: {
+    selected(): Version | null {
+      return this.selected_version ? this.extension.versions[this.selected_version] : null
+    },
+    compiled_markdown(): string {
+      if (!this.selected?.readme) {
+        return 'No readme available'
+      }
+      return marked(this.selected.readme, { sanitize: true })
+    },
+    available_tags(): string[] {
+      return Object.keys(this.extension?.versions ?? [])
+    },
+    permissions(): (undefined | string) {
+      if (!this.selected_version) {
+        return 'Select a version'
+      }
+      const versions = this.extension?.versions
+      if (versions && this.selected_version in versions) {
+        return versions[this.selected_version].permissions
+      }
+      return 'No permissions required'
+    },
+  },
+  watch: {
+    extension() {
+      const [first] = Object.keys(this.extension.versions) ?? ''
+      this.selected_version = first
```

Comment:
You can unpack directly on the variable 
```suggestion
[this.selected_version] = Object.keys(this.extension.versions) ?? ''
```

Reply:
this doesnt work for some reason...

```
ERROR in [eslint] 
/home/will/sambashare/companion-docker/core/frontend/src/components/kraken/ExtensionModal.vue
  191:13  error  Parsing error: Array element destructuring pattern expected


```